### DISCUSSION TITLE
DBeaver needs JVM 17--where to get for Ubuntu?

### DISCUSSION THREAD

**Discussion by mpemburn** [dbeaver/dbeaver#19776] [Q&A]
Hi,

I'm trying to run DBeaver on Ubuntu 22, which is running in a virtual machine on my MacBook with the M1 chip.  It has Java 11.0.18 installed, but DBeaver says this is "unsuitable" and wants JVM 17.  I haven't found any way to update to this version.  Can you advise?

Thanks!

Mark

**Comment by chirontt** [comment]
Can't you install the Java 17 version in your virtual machine, using the command: `sudo apt install openjdk-17-jre`?

Or try the [SDKMAN](https://sdkman.io/) tool to install, manage and use all the JDK/JRE of any versions in your Linux system. You don't need to depend on the versions provided by Ubuntu .

**Reply by mpemburn** [reply]
Hi @chirontt,

Took me a while to get around to it, but that absolutely worked--thanks!

Mark
